### PR TITLE
 [patch][bug] CEECORE-644

### PR DESCRIPTION
**Pull request by scrajesh**
* AutoPrefixer and PostCSS-PresetEnv uses .browserlistrc
* auto generate .browserlistrc when scaffolding a new wml-app

> Browsers set in the config file can be validated using following:
   * npx browserslist
   * npx autoprefixer --info

### CODE REVIEW

File: packages/electrode-archetype-react-app-dev/.browserslistrc [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Browsers that we support
+
+last 2 versions
+ie >= 9
+> 5%
```
Comment (scrajesh) [inline packages/electrode-archetype-react-app-dev/.browserslistrc]: Not sure if this file is required? As this doesn't help in generating the .browserlistrc file.

File: packages/electrode-archetype-react-app/arch-clap.js [hunk 2]
```diff
@@ -267,6 +267,25 @@ function startAppServer(options) {
...
 }
 
+function generateBrowsersListRc() {
+  const destnDir = Path.resolve(archetype.AppMode.src.client);
+  if (!Fs.existsSync(destnDir)) return;
+
+  const configRcFile = ".browserslistrc";
+  const destnRc = Path.join(destnDir, configRcFile);
+  if (Fs.existsSync(destnRc)) {
+    logger.info(`You've ${configRcFile} config file already. It should be removed to auto generate.`);
```
Comment (jchip) [inline packages/electrode-archetype-react-app/arch-clap.js]: does this gets logged even if the file was auto generated by electrode?

Comment (scrajesh) [reply packages/electrode-archetype-react-app/arch-clap.js]: Yes, irrespective of generated and custom file.

File: packages/electrode-archetype-react-app/arch-clap.js [hunk 3]
```diff
@@ -267,6 +267,25 @@ function startAppServer(options) {
   return exec(`node`, options, Path.join(archetype.AppMode.lib.server, "index.js"));
 }
 
+function generateBrowsersListRc() {
+  const destnDir = Path.resolve(archetype.AppMode.src.dir, '../');
```
Comment (scrajesh) [inline packages/electrode-archetype-react-app/arch-clap.js]: I assume this way of getting the project root directory works as expected across platforms.

File: packages/electrode-archetype-react-app/arch-clap.js (hunk 2 above)
Comment (scrajesh) [reply packages/electrode-archetype-react-app/arch-clap.js]: Modified this part to check for custom props if any and warn the user about it.
If not simply generating the config file with the default props.

**Pull request merged**